src/components/Frame/components/ContextualSaveBar/tests/ContextualSaveBar.test.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
-import {mountWithAppProvider, trigger} from 'test-utilities/legacy';
+import {act, mountWithAppProvider, trigger} from 'test-utilities/legacy';
 import {Button, Image} from 'components';
-import ContextualSaveBar from '../ContextualSaveBar';
+import {ContextualSaveBar} from '..';
```

Comment:
```suggestion
import {ContextualSaveBar} from '../ContextualSaveBar';
```

Reply:
there are few other places that this occurred:

https://github.com/Shopify/polaris-react/blob/cda89db560c4a5d67045db888abf60978faf2c11/src/components/Backdrop/tests/Backdrop.test.tsx#L3
https://github.com/Shopify/polaris-react/blob/cda89db560c4a5d67045db888abf60978faf2c11/src/components/Banner/tests/Banner.test.tsx#L14
https://github.com/Shopify/polaris-react/blob/cda89db560c4a5d67045db888abf60978faf2c11/src/components/PageActions/tests/PageActions.test.tsx#L3
...

do you want me to send a PR for those too?

Reply:
That would be great. Thanks for doing this!